GETTING_STARTED.md

```diff
@@ -1018,6 +1018,14 @@ To set the attributes for each of the factories, you can pass in a hash as you n
 twenty_year_olds = build_list(:user, 25, date_of_birth: 20.years.ago)
 ```
 
+There are `*_list_with_index` methods for using the index to set different attribute values for each created item:
+
+```ruby
+twenty_somethings = build_list(:user, 10) do |user, i|
```

Comment:
The text says `*_list_with_index`, but the code example uses `build_list`. I think I like what it is in the code example a bit better (see the next comment).